test/federated/test_train_config.py

```diff
@@ -75,20 +78,22 @@ def forward(self, x):
     assert loss_after < loss_before
 
 
-@pytest.mark.skip(reason="bug in pytorch version 1.1.0, jit.trace returns raw C function")
+@pytest.mark.skipif(
+    torch.__version__ >= "1.1",
+    reason="bug in pytorch version 1.1.0, jit.trace returns raw C function",
+)
 def test_train_config_with_jit_trace(hook, workers):  # pragma: no cover
     alice = workers["alice"]
-    me = workers["me"]
 
     data = torch.tensor([[-1, 2.0], [0, 1.1], [-1, 2.1], [0, 1.2]], requires_grad=True)
     target = torch.tensor([[1], [0], [1], [0]])
 
     dataset = sy.BaseDataset(data, target)
-    alice.add_dataset(dataset, key="vectors")
+    alice.add_dataset(dataset, key="gaussian_mixture")
 
     @hook.torch.jit.script
-    def loss_fn(real, pred):
-        return ((real.float() - pred.float()) ** 2).mean()
+    def loss_fn(pred, target):
```

Comment:
Should the order be changed here (I prefer the new ordering of pred and target but want to be sure this does not break anything)